tests/regression_tests/unstructured_mesh/source_sampling/test.py

```diff
@@ -0,0 +1,114 @@
+from itertools import product
+
+from pathlib import Path
+import pytest
+import numpy as np
+
+import openmc
+import openmc.lib
+
+from tests import cdtemp
+from tests.testing_harness import PyAPITestHarness
+
+TETS_PER_VOXEL = 12
+
+# This test uses a geometry file with cells that match a regular mesh. Each cell
+# in the geometry corresponds to 12 tetrahedra in the unstructured mesh file.
+@pytest.fixture
+def model():
+    openmc.reset_auto_ids()
+
+    ### Materials ###
+    materials = openmc.Materials()
+
+    water_mat = openmc.Material(material_id=3, name="water")

```

Comment:
Is it really necessary to assign an ID manually?

Reply:
Nope. Removing...